edx_name_affirmation/models.py

```diff
@@ -11,6 +12,32 @@
 User = get_user_model()
 
 
+class VerifiedNameStatus(str, Enum):
+    """
+    Possible states for the verified name.
+
+    Pending: the verified name has been created, but not seen by anyone
```

Comment:
Probably I should just take this out and say pending is the start/default status because trying to be specific about what hasn't happened is going to duplicate what the later statuses say.

Reply:
4295b59